.travis.yml

```diff
@@ -49,6 +49,18 @@ jobs:
     - env: SCRIPT=test:fastboot
     - env: SCRIPT="test:one ember-canary"
 
+    - stage: 'Deploy'
```

Comment:
I think the default stages are `test` and `deploy`. I'm not sure if this will work with a capital `D`.

Reply:
Can you explain the reasons for your concern further? The other jobs in the staged pipeline setup in this file also work with the arbirtrary name setup and I'm not sure what the `deploy` stage name is needed for additionally

Reply:
hmm interesting, I must have been blind... 😅 

I'm wondering if it just uses the implicit job order in that case 🤔